layout/layout_test.go

```diff
@@ -1224,3 +1147,431 @@ func testImage(t *testing.T, when spec.G, it spec.S) {
 		})
 	})
 }
+
+func testImageIndex(t *testing.T, when spec.G, it spec.S) {
```

Comment:
Organizational nit: should this go in its own file?

Reply:
I tried to move this to another file, but there is work to do to avoid sharing the docker registry

Reply:
Another comment, many of the tests are pulling directly from Docker Hub, which causes us to hit the rate limit fairly easily. Something to possibly refactor

Reply:
I believe this was fixed by @jjbustamante

Reply:
Yes, I think I moved the test to `index_test.go`